### DISCUSSION TITLE
How to open Neovide from terminal without blocking the terminal?

### DISCUSSION THREAD

**Discussion by thatanjan** [neovide/neovide#2747] [Q&A] [answered]
When I open neovide from the terminal it just blocks the terminal. For example: 
```
neovide cules-coding/
```
I get this:
![image](https://github.com/user-attachments/assets/43f4703a-c593-4b7a-9b8a-94e2a86c5f65)

Thanks in advance 🙏

**Comment by Mixaz017** [comment, marked as answer]
Append the `--fork` flag. You could also set this in the [config file](https://neovide.dev/config-file.html) if you want this to be the default behavior.

**Reply by thatanjan** [reply]
Thank you so much @Mixaz017 🙏
